orc8r/lib/go/definitions/const.go

```diff
@@ -23,4 +23,6 @@ const (
 	MetricsdServiceName     = "metricsd"
 	StateServiceName        = "state"
 	StreamerServiceName     = "streamer"
+
+	SQLiteDriver = "sqlite3"
```

Comment:
Just use https://github.com/magma/magma/blob/6aaa8e0f705fc7957dd89ccba3dc2b2c08dce294/orc8r/cloud/go/sqorc/open.go#L30 instead of creating a new var

Reply:
oh nice didn't see that